packages/tools/viewer-alpha/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./viewer";

```

Comment:
are we planning a UMD version of the viewer? do we still want to use index.ts that exports everything?

Reply:
Talked offline, but the plan is to not initially have UMD or even browser consumable modules. The first pass will be having a single bundled ES6 module (which will become module***s*** once we start using dynamic imports) with @babylonjs/* dependencies declared as external so they are not bundled. This will require a top level index.ts and will allow us to not "ship our directory structure" (we don't want to have to maintain the source directory structure for back compat). Later we will include bundled ES6 modules that are self contained (no external dependencies) which can be used directly in the browser without any build/bundler.